testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -1,14 +1,11 @@
-use std::cmp;
+use std::convert::From;
```

Comment:
Not necessary, already imported by `std::prelude::v1`

Reply:
Sure